compsat/ls-ganap
# Refactor/dockerize client

Dockerized application client

Testing Steps: 
In the ls-ganap directory, run ff commands: 
```
docker-compose build
docker-compose up
```

make sure to get the latest .env files for the api and client
